src/app/components/add-movie/add-movie-page.component.spec.ts

```diff
@@ -1,14 +1,17 @@
 import {ComponentFixture, TestBed} from '@angular/core/testing';
 
 import {AddMoviePageComponent} from './add-movie-page.component';
+import {HttpClientModule} from "@angular/common/http";
+import {FormsModule} from "@angular/forms";
 
 describe('AddMovieComponent', () => {
```

Comment:
you need to add tests for adding movies, it's like the main goal the component does. You're gonna need to mock services and check if it was called, and with what payload it was called